winsw.csproj

```diff
@@ -92,6 +92,7 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <None Include="packages.config" />

```

Comment:
Once done properly, this change should also disappear since this is  the only Nuget Package you are using.